.github/CONTRIBUTING.md

```diff
@@ -1,112 +1,51 @@
+There are many ways to contribute to the project!
 
-Feature request issues will remain closed until we see sufficient interest via comments and [👍 reactions](https://help.github.com/articles/about-discussions-in-issues-and-pull-requests/) from the community.
-
-You can see a [list of current feature requests which require votes here](https://github.com/woocommerce/woocommerce/issues?q=label%3A%22votes+needed%22+label%3Aenhancement+sort%3Areactions-%2B1-desc+is%3Aclosed).
-
-## Technical Support / Questions ❓
-
-We don't offer technical support on GitHub so we recommend using the following:
-
-**Reading our documentation**
-Usage docs can be found here: https://docs.woocommerce.com/
-
-If you have a problem, you may want to start with the self help guide here: https://docs.woocommerce.com/document/woocommerce-self-service-guide/
-
-**Technical support for premium extensions or if you're a WooCommerce.com customer**
- from a human being - submit a ticket via the helpdesk
-https://woocommerce.com/contact-us/ 
-
-**General usage and development questions**
-- WooCommerce Slack Community: https://woocommerce.com/community-slack/
-- WordPress.org Forums: https://wordpress.org/support/plugin/woocommerce
-- The WooCommerce Help and Share Facebook group
-
-**Customizations**
-- [WooExperts](https://woocommerce.com/experts/)
-- [Codeable](https://codeable.io/)
+- [Translating strings into your language](https://github.com/woocommerce/woocommerce/wiki/Translating-WooCommerce).
+- Answering questions on the various WooCommerce communities like the [WP.org support forums](https://wordpress.org/support/plugin/woocommerce/).
+- Testing open [issues](https://github.com/woocommerce/woocommerce/issues) or [pull requests](https://github.com/woocommerce/woocommerce/pulls) and sharing your findings in a comment.
+- Testing WooCommerce beta versions and release candidates. Those are announced in the [WooCommerce development blog](https://woocommerce.wordpress.com/).
+- Submitting fixes, improvements, and enhancements.
+- To disclose a security issue to our team, [please submit a report via HackerOne](https://hackerone.com/automattic/).
 
-## Build process 🛠
+If you wish to contribute code, before [forking](https://help.github.com/articles/fork-a-repo/) WooCommerce, commiting your changes,
+and [sendind a pull request](https://help.github.com/articles/using-pull-requests/), please see the information in the sections below.
 
-The source code found on GitHub does not contain compiled CSS or Javascript, and is missing some functionality such as Gutenberg Blocks and the REST API, until the build process is ran.
+WooCommerce is licensed under the GPLv2+, and all contributions to the project will be released under the same license. You maintain copyright over any contribution you make, and by submitting a pull request, you are agreeing to release that contribution under the GPLv2+ license.
```

Comment:
Our license.txt says GPLv3+

Reply:
I just pushed a commit to fix that. Thanks